lumigator/frontend/src/components/views/Settings.vue

```diff
@@ -0,0 +1,354 @@
+<template>
+  <div class="settings-container">
+    <div>
+      <h3 class="settings-title">Settings</h3>
+    </div>
+    <div class="api-keys-container">
+      <div class="api-keys-header">
+        <h4 class="api-keys-title">API Keys</h4>
+        <p class="api-keys-description">
+          To use API-based models in Lumigator, add your API keys.
+          <a
+            href="https://mozilla-ai.github.io/lumigator/operations-guide/configuration.html#api-settings"
+            class="learn-more-link"
+            rel="noopener"
+            target="_blank"
+            >Learn more
```

Comment:
This the styling we currently use for external links:

```suggestion
            href="https://mozilla-ai.github.io/lumigator/operations-guide/configuration.html#api-settings"
            target="_blank"
            style="background-color: transparent"
            >Learn more <span class="pi pi-arrow-up-right"
```